third_party/rvi_lib/CMakeLists.txt

```diff
@@ -1,7 +1,17 @@
-add_subdirectory(libjwt)
 add_library(rvi STATIC
 	src/btree.c src/rvi.c src/rvi_list.c
 )
 target_include_directories(rvi PUBLIC include)
 target_include_directories(rvi PRIVATE libjwt/include)
-target_link_libraries(rvi jwt_static crypto ssl jansson)
+target_link_libraries(rvi libjwt_static crypto ssl jansson)
+
+include(ExternalProject)
+ExternalProject_Add(libjwt
+    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/libjwt
+    CONFIGURE_COMMAND autoreconf -i ${CMAKE_CURRENT_SOURCE_DIR}/libjwt && ${CMAKE_CURRENT_SOURCE_DIR}/libjwt/configure --prefix=<INSTALL_DIR>
+    INSTALL_DIR ${PROJECT_BINARY_DIR}
+    BUILD_COMMAND ${MAKE})
```

Comment:
Unfortunately ExternalProject doesn't play nicely with Yocto builds--we had to make a bunch of changes (see git log) to get this to work cleanly.  This really should all be built in a single cmake environment, rather than shelling out to separate build commands.